### ISSUE TITLE
Unnecessary prompt

### ISSUE THREAD

**Issue by Matt321456** [sawhney17/logseq-smartblocks#50]
Hi, great plugin! What's the issue with that unnecessary prompt: "Would you like to insert with inputs?". It's quite obvious to me that the answer is yes.. Please help remove these unnecessary steps when speed is the whole point. Thanks

**Comment by ScriptingDad** [comment]
I agree with this.  I think maybe a more elegant way to handle this would be to show the inputs.  If nothing is entered then the submit renders with those inputs missing.  And "Go back" could be Cancel to just stop the input altogether, which is what I believe it does. 

I for one agree with less clicks whenever possible.
